released/packages/coq-bignums/coq-bignums.8.12.0/opam

```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+maintainer: "Vincent.Laporte@inria.fr"
+homepage: "https://github.com/coq/bignums"
+dev-repo: "git+https://github.com/coq/bignums.git"
+bug-reports: "https://github.com/coq/bignums/issues"
+authors: [
+  "Laurent Théry"
+  "Benjamin Grégoire"
+  "Arnaud Spiwack"
+  "Evgeny Makarov"
+  "Pierre Letouzey"
+]
+license: "LGPL-2.1"
```

Comment:
```suggestion
license: "LGPL-2.1-only"
```
Unfortunately, this does not currently pinpoint an exact license, and file headers are not helpful. From the commit history, it seems the intent was to license it under Coq's license, which is LGPL-2.1 *only*, i.e., it can't be licensed under LGPL-3.0.

Reply:
Good catch.
Actually `extra-dev/packages/coq-bignums/coq-bignums.8.12.dev/opam` was already fine, but not all `coq-bignums*/opam` files.
Should I fix only the opam file of this PR, or all opam versions accordingly?

Reply:
Maybe you can do a separate PR with the other opam versions? Also, feel free to use the commit suggestion here as a way to trigger a rebuild when `opam update` finally gives the expected result...

Reply:
OK, let's do that, thanks!